citylearn/electric_vehicle_charger.py

```diff
@@ -0,0 +1,374 @@
+import inspect
+from typing import List, Dict
+from citylearn.base import Environment
+
+from citylearn.electric_vehicle import electric_vehicle
+
+ZERO_DIVISION_CAPACITY = 0.00001
+
+
+class Charger(Environment):
+    def __init__(
+            self,
+            nominal_power: float,
+            efficiency: float = None,
+            charger_id: str = None,
+            charger_type: int = None,
+            max_charging_power: float = 50.0,
+            min_charging_power: float = 0.0,
+            max_discharging_power: float = 50.0,
+            min_discharging_power: float = 0.0,
```

Comment:
`max_charging_power`, `min_charging_power`, `max_discharging_power`, and `min_discharging_power` should have their default values defined in their `setter` function not the `init` function. See how it is done in other classes in `citylearn.energy_model`.